src/android/SplashScreen.java

```diff
@@ -194,9 +263,25 @@ public void run() {
                 // TODO: Use the background color of the webView's parent instead of using the
                 // preference.
                 root.setBackgroundColor(preferences.getInteger("backgroundColor", Color.BLACK));
+                
                 root.setLayoutParams(new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT,
                         ViewGroup.LayoutParams.MATCH_PARENT, 0.0F));
-                root.setBackgroundResource(drawableId);
+                
+                // Use an ImageView to render the image because of its flexible scaling options.
+                splashImageView = new ImageView(context);
+                splashImageView.setImageResource(drawableId);
```

Comment:
Hmm, I figured the background would show for a square image when SplashMaintainAspectRatio is set. I think you've done due diligence to it though, so if it's not required, then don't even bother setting it.